### PR TITLE
New Port Mezzer

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Mezzer

## Game Information
- **Title**: Mezzer
- **URL**: https://danieldavis.itch.io/mezzer

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Cebion — APPROVED** [review]

File: ports/mezzer/port.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "mezzer.zip",
+  "items": [
+    "MEZZER",
```
Comment (Cebion) [inline ports/mezzer/port.json:5]:
```suggestion
    "MEZZER.sh",
```

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by cdeletre — DISMISSED** [review]

File: ports/mezzer/MEZZER.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+fi
+
+$GPTOKEYB "gmloader" &
+
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+pm_platform_helper "$GAMEDIR/gmloader"
+./gmloader mezzer.apk
+
+pm_finish
+printf "\033c" > /dev/tty0
```
Comment (Cebion) [inline ports/mezzer/MEZZER.sh:115]: no longer used
```suggestion
printf "\033c" > /dev/tty0
```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/mezzer/MEZZER.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    if [ "$checksum" = "02cbf4cf97b55391e3c6105cfd70fa11" ]; then
+        pm_message "data.win is being patched..." && \
+        $ESUDO $controlfolder/xdelta3 -d -s gamedata/data.win -f ./patch/mezzer.xdelta gamedata/game.droid && \
+        rm gamedata/data.win
+    else
+        pm_message "Error: MD5 checksum of data.win does not match any expected version."
+        exit 1
+    fi
+else    
+    echo "data.win not in gamedata folder or game has been patched."
```
Comment (Cebion) [inline ports/mezzer/MEZZER.sh:88]:
```suggestion
    pm_message "data.win not in gamedata folder or game has been patched."
```

File: ports/mezzer/MEZZER.sh [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            rm -f "$GAMEDIR/gamedata/Mezzer.exe"
+            rm -f "$GAMEDIR/gamedata/D3DX9_43.dll"
+	    rm -f "$GAMEDIR/Mezzer.exe"
+        fi
+    else
+        pm_message "Error: MD5 checksum of Mezzer.exe does not match the expected checksum."
+        exit 1
+    fi
+else
+    echo "Mezzer.exe not detected in /mezzer"
```
Comment (Cebion) [inline ports/mezzer/MEZZER.sh:72]:
```suggestion
    pm_message "Mezzer.exe not detected in /mezzer"
```

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**